dspace/config/registries/schema-periodical-types.xml

```diff
@@ -0,0 +1,50 @@
+<dspace-dc-types>
+
+    <dspace-header>
+        <title>Periodical Types</title>
+    </dspace-header>
+
+    <dc-schema>
+        <name>periodical</name>
+        <namespace>https://schema.org/Periodical</namespace>
+    </dc-schema>
+
+    <dc-schema>
+        <name>creativework</name>
+        <namespace>https://schema.org/CreativeWork</namespace>
+    </dc-schema>
+
+    <dc-schema>
+        <name>creativeworkseries</name>
+        <namespace>https://schema.org/CreativeWorkSeries</namespace>
+    </dc-schema>
+
+    <!--NO EQUIVALENT FOUND
+        - journal.title
```

Comment:
Use dc.title?

Reply:
This is indeed configured to use dc.title in this PR: https://github.com/DSpace/DSpace/blob/65c3039c1dfb7d26f0139306785e86aa5ab9a73a/dspace/config/submission-forms.xml#L440